apps/demos/Demos/Chat/Customization/jQuery/index.js

```diff
@@ -3,57 +3,57 @@ $(() => {
     height: 710,
     items: messages,
     user: currentUser,
-    dayHeaderFormat: dayHeaderFormat[0],
+    dayHeaderFormat: dayHeaderFormats[0],
+    messageTimestampFormat: messageTimestampFormats[0],
     onMessageEntered({ component, message }) {
       component.renderMessage(message);
     },
-    messageTimestampFormat: messageTimestampFormat[0],
   }).dxChat('instance');
 
   $('#show-avatar').dxCheckBox({
     value: true,
-    text: 'Show Avatar',
+    text: 'Avatar',
     onValueChanged(data) {
       chat.option('showAvatar', data.value);
     },
   });
 
   $('#show-user-name').dxCheckBox({
     value: true,
-    text: 'Show User Name',
+    text: 'User Name',
     onValueChanged(data) {
       chat.option('showUserName', data.value);
     },
   });
 
   $('#show-day-headers').dxCheckBox({
     value: true,
-    text: 'Show Day Headers',
+    text: 'Day Headers',
```

Comment:
In all other demos you use 'Day Header'. Should we use this text in other demos as well?

Reply:
[3135449](https://github.com/DevExpress/DevExtreme/pull/28482/commits/3135449967fc6025f9a2df23c45bf7a58b797228)